### DISCUSSION TITLE
How to publish GitHub release?

### DISCUSSION THREAD

**Discussion by sualeh** [jreleaser/jreleaser#1276] [Q&A]
@aalmiray I have just started using JReleaser to create my GitHub releases. It seems to be working except the last step - to actually publish the release. How do I get my GitHub release published?

**Comment by aalmiray** [comment]
What seems to be the problem? Do you have a stacktrace or a copy of `trace.log`?
Did you follow the [checklist](https://jreleaser.org/guide/latest/checklist.html)?

Running the build locally in `--dry-run` mode can help solving any issues you may encounter. This lets you verify the configuration is working as expected.

**Comment by sualeh** [comment]
@aalmiray I did follow the checklist, and I also did a dry run. The issue is this: when JReleaser overwrites a GitHub release, it does everything except publishing it. However, when creating a new release, JReleaser publishes it as expected. Please see the [JReleaser configuration](https://github.com/schemacrawler/SchemaCrawler-Installers/blob/main/jreleaser.yml) and [attached trace logs](https://github.com/jreleaser/jreleaser/files/11006623/windows-jreleaser-out.zip).
